src/platforms/java/common/configuration/integrations/graphql.mdx

```diff
@@ -10,92 +10,92 @@ notSupported:
 
 Sentry GraphQL integration provides an integration with [GraphQL Java](https://github.com/graphql-java/graphql-java/) through:
 
-- `SentryDataFetcherExceptionHandler` which captures exceptions thrown during data fetcher executions.
-- `SentryInstrumentation` which creates spans around each data fetcher execution.
+- `SentryGenericDataFetcherExceptionHandler` which checks for exceptions thrown during data fetcher executions and passes them to `SentryInstrumentation`.
+- `SentryInstrumentation` which creates spans around each data fetcher execution, captures exceptions and adds breadcrumbs.
+
+If you're using `spring-graphql`, we offer automatic configuration of our GraphQL integration when using our `sentry-spring-boot-starter` or `sentry-spring-boot-jakarta-starter` integrations.
 
 ## Install
 
+```groovy {tabTitle:Gradle Plugin}
+plugins {
+  id "io.sentry.jvm.gradle" version "{{@inject packages.version('sentry.java.android.gradle-plugin', '3.12.0') }}"
+}
+```
+
+```groovy {tabTitle:Gradle}
+implementation 'io.sentry:sentry-graphql:{{@inject packages.version('sentry.java.graphql', '6.28.0') }}'
+```
+
 ```xml {tabTitle:Maven}
 <dependency>
     <groupId>io.sentry</groupId>
     <artifactId>sentry-graphql</artifactId>
-    <version>{{@inject packages.version('sentry.java.graphql', '5.4.0') }}</version>
+    <version>{{@inject packages.version('sentry.java.graphql', '6.28.0') }}</version>
 </dependency>
 ```
 
-```groovy {tabTitle:Gradle}
-implementation 'io.sentry:sentry-graphql:{{@inject packages.version('sentry.java.graphql', '5.4.0') }}'
-```
-
 ```scala {tabTitle: SBT}
-libraryDependencies += "io.sentry" % "sentry-graphql" % "{{@inject packages.version('sentry.java.graphql', '5.4.0') }}"
+libraryDependencies += "io.sentry" % "sentry-graphql" % "{{@inject packages.version('sentry.java.graphql', '6.28.0') }}"
 ```
 
 For other dependency managers, check out the [central Maven repository](https://search.maven.org/artifact/io.sentry/sentry-graphql).
 
-## Capture Exceptions
+## Setup
 
-`SentryDataFetcherExceptionHandler` captures the exception, sends it to Sentry, and calls the delegate responsible for handling the exception.
-
-Create a new instance of `SentryDataFetcherExceptionHandler`, pass the delegate that handles the exception, and configure `defaultDataFetcherExceptionHandler` when building a `GraphQL` instance:
+Create a new instance of `SentryGenericDataFetcherExceptionHandler`, pass the delegate that handles the exception, and configure `defaultDataFetcherExceptionHandler` when building a `GraphQL` instance.
+Create a new instance of `SentryInstrumentation` and configure `instrumentation` when building a `GraphQL` instance:
```

Comment:
- [ ] Link to filtering docs
Came up in https://github.com/getsentry/sentry-java/discussions/2897